apiserver/facades/agent/provisioner/imagemetadata_test.go

```diff
@@ -60,7 +60,7 @@ func (s *ImageMetadataSuite) TestMetadataNone(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 
 	expected := make([][]params.CloudImageMetadata, len(s.machines))
-	for i, _ := range result.Results {
```

Comment:
I know a lot of tools like to do this, but I'm not sold that it is *better* for human understanding. Very specifically, a lot of times Go's handling of "what is the 0th vs 1st" parameter in automatic range loops means that it is actually clearer for a human to read if it has the extra '_'. It makes it clear to a human that
a) I'm only looking at the index and
b) explicitly not looking at the value.

Especially given that in something like python:
 for foo in ['a', 'b', 'c']:
    print(foo)
is 'a', 'b', 'c', not 0, 1, 2.

I'd rather not have encoded a "though must not use i, _" in range expressions.
Is it easy to turn it off?
Do other people feel like they find the 2 argument form ugly, or is it just that the simplify linter knows "it isn't needed".
*I* feel like it is true that the Language is unambiguous, but humans are not perfect readers of Language.

Thoughts?

Reply:
Let me have a look at any possible options. I personally find it a tiny bit easier to read without `_`, but again it's a matter of taste.

Reply:
Unfortunately the std gofmt tool doesn't allow us to turn this off when using simplify re-writing: https://github.com/golang/go/blob/7d704a97608292150263f34da9430ac70a8c4ffd/src/cmd/gofmt/simplify.go#L86-L97

Reply:
I agree with @rogpeppe here. We are using go, and less people have python paged into working memory. I'd rather lose the ", _" as well.